apps/chat/src/components/Chat/Publish/ApproveRequiredSection.tsx

```diff
@@ -171,8 +175,12 @@ export const ApproveRequiredSection = ({
 
   return (
     <CollapsibleSection
-      name={t(name)}
-      openByDefault={openByDefault}
+      onToggle={handleToggle}
+      name={name}
+      openByDefault={
+        openByDefault ??
+        !collapsedSections.includes(PUBLISHING_APPROVE_REQUIRED_NAME)
```

Comment:
Why don't we do it inside `useSectionToggle` hook itself and return `openedByDefault` instead of `collapsedSections`?
```suggestion
        !collapsedSections.includes(name)
```